vmware-base/src/main/java/com/cloud/hypervisor/vmware/util/VmwareHelper.java

```diff
@@ -549,12 +549,6 @@ public static void setBasicVmConfig(VirtualMachineConfigSpec vmConfig, int cpuCo
 
         cpuInfo.setReservation((long)cpuReservedMhz);
         vmConfig.setCpuAllocation(cpuInfo);
-        if (cpuSpeedMHz != cpuReservedMhz) {
-            vmConfig.setCpuHotAddEnabled(true);
-        }
-        if (memoryMB != memoryReserveMB) {
-            vmConfig.setMemoryHotAddEnabled(true);
-        }
```

Comment:
thsi removal does not seem relevant to your feature description. can you explain @DK101010 ?

Reply:
@DaanHoogland sorry for my late answer. In my opinion I think yes and no. 
Hot add memory and cpu will be enabled when reserved memory and cpu is in use. But this is not correct. Both features are independent, therefore I thought it is good to remove it in this PR. But if you want I can create a separate PR for this commit.

Reply:
not sure if that is needed and/or suffices; How about when people want to hot-add and don't care about reservation?

Reply:
@DaanHoogland hot add and reservation are two different features and they are independent of each other. I talked with my team and test it in my environment to get sure and in my env it works. In my point of view this code lines makes no sense.

Reply:
ok @DK101010 , if that's verified I'm fine with the code.

Reply:
I have not much experience with the codebase for VMware to comment on this piece of code. Good that it has been verified. I will rely on the contributor tests.

Reply:
I've my doubts on these changes, pinged vmware experts @harikrishna-patnala @nvazquez @sureshanaparti

Reply:
@DK101010 I think removing these lines will cause a regression on cpu and memory overcommit, have you tested those cases?

Reply:
Hi @nvazquez not sure what do you mean with regression.
Reservation and hot add are two different things. I can use reservation in it doesn't matter if hot add enabled/disbaled and it makes for me not sense to enable this when I'm reserve memory .
Also you can't more reserve as you have for the current vm. 

I may be missing something, but currently I don't see a problem here.